source/service/oneke.rst

```diff
@@ -413,7 +413,7 @@ Traefik is deployed during the cluster creation from an official Helm chart with
 
 - A dedicated namespace ``traefik-system`` is provided.
 - An `anti-affinity <https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity>`_ rule is applied to Traefik pods to minmize potential downtime during failures and upgrades.
-- Traefik is exposed on a ``NodePort`` type of the `Kubernetes Service <https://kubernetes.io/docs/concepts/services-networking/service/>`_. By default HAProxy instance (running on the leader VNF node) connects to all worker nodes to ports ``32080`` and ``32443``, then forwards all traffic comming to HAProxy to ports ``80`` and ``443`` to the Traefik instance (running inside Kubernetes).
+- Traefik is exposed on a ``NodePort`` type of the `Kubernetes Service <https://kubernetes.io/docs/concepts/services-networking/service/>`_. By default HAProxy instance (running on the leader VNF node) connects to all worker nodes to ports ``32080`` and ``32443``, then forwards all traffic coming to HAProxy to ports ``80`` and ``443``, to the Traefik instance (running inside Kubernetes).
```

Comment:
@Marc-Ripley if the original sentence sounds illogical, then maybe this one sounds better :thinking: :
```suggestion
- Traefik is exposed on a ``NodePort`` type of the `Kubernetes Service <https://kubernetes.io/docs/concepts/services-networking/service/>`_. By default HAProxy instance (running on the leader VNF node) connects to worker node ports ``32080`` and ``32443``, then forwards all traffic coming to HAProxy (ports ``80`` and ``443``), to the Traefik instance (running inside Kubernetes).
```